styles.css

```diff
@@ -0,0 +1,872 @@
+/*    border: 3px solid greenyellow;*/
+/*}*/
+
+/*.navigation,*/
+/*.header___hero-section,*/
+/*.header___squiggle-line,*/
+/*.main__paragraph--about-section,*/
+/*.main__paragraph--cta-line,*/
+/*#main__content-section {*/
+/*    border: 3px solid yellow;*/
+/*}*/
+
+/*.main__content-article,*/
+/*.main__content-article--image,*/
+/*.main__content-article--block*/
+/*{*/
+/*    border: 3px solid blue;*/
+/*}*/
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+body {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 400;
+}
+
+body a {
+    text-decoration: none;
+    color: var(--main-focus-color);
+}
+
+h1,
+h2,
+h3,
+h4 {
+    font-family: 'Merriweather', serif;
+}
+
+h1 {
+    /*margin: 20px;*/
+    margin: 5px;
+    font-size: 70px;
+}
+
+h1::first-letter {
+    text-transform: uppercase;
+}
+
+h3::first-letter {
+    text-transform: uppercase;
+}
+
+nav {
+    text-transform: capitalize;
+}
+
+button {
+    font-family: inherit;
+}
+
+h2 {
+    margin: 10px 0 10px 0;
+}
+
+h4 {
+    color: var(--main-focus-color);
+    font-size: 18px;
+}
+
+.navigation__list-items {
+    text-align: center;
+}
+
+.navigation___link {
+    color: var(--main-font-color-light);
+    font-weight: 700;
+    font-size: 32px;
+    text-decoration: none;
+}
+
+.main__paragraph--h2 {
+    font-size: 42px;
+}
+
+.main__paragraph--cta-line {
+    font-size: 35px;
+}
+
+.main__content-article--header {
+    text-transform: capitalize;
+    font-size: 60px;
+}
+
+.main__content-article--paragraph {
+    font-size: 40px;
+}
+
+.content-article--link {
+    display: block;
+    margin-top: 100px;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 700;
+    font-size: 30px;
+    letter-spacing: 1px;
+}
+
+.footer__copyright-paragraph {
+    text-align: center;
+    color: var(--main-font-color-light);
+    font-size: 10px;
+    font-weight: 300;
+    letter-spacing: 1px;
+}
+
+.services__block--paragraph {
+    color: var(--main-font-color-medium);
+    margin-top: 5em;
+    font-size: 13px;
+}
+
+.footer__form--header {
+    color: var(--main-font-color-light);
+    font-size: 4.5em;
+    margin: 0.5em;
+    text-align: center;
+}
+
+.footer__form--paragraph {
+    text-align: center;
+    color: var(--main-font-color-light);
+    font-weight: 300;
+    font-size: 40px;
+    margin: 1em 0;
+}
+
+.footer__form--form-section {
+    color: var(--main-font-color-light);
+    font-size: 30px;
+}
+
+.form__input[type=text],
+.form__input--text-area {
+    font-family: Roboto, sans-serif;
+    font-size: 30px;
+}
+
+.footer__copyright-paragraph {
+    font-size: 25px;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+.main__inner-container a:link{
+    color: var(--main-focus-color);
+}
+
+.main__inner-container a:visited{
+    color: var(--main-shadow-focus-color);
+}
+
+.main__outer-container {
```

Comment:
Een kleine aanvulling: je had hier een reusable van kunnen maken met display: flex, width: 100%, flex-direction: row en justify-content: center. Dan heb je al een hele stevige parent container. De position relative had je in dit geval ook kunnen koppelen aan de header! (Dit kan je dan bijvoorbeeld door met een id). Dit voelt misschien een beetje omslachtig, maar nu heb jij meerdere declaraties van die outer container, terwijl we eigenlijk wel er voor willen zorgen dat die parent container niet te veel afwijkt van de rest van de pagina. We zorgen dus met een reusable zoals dat voor meer structuur. En dan kunnen we altijd nog op de header of main, of footer zelf nog andere styling regels instellen.